oidc/pkce.ml

```diff
@@ -0,0 +1,61 @@
+(**
+ * Copyright 2022 Ulrik Strid. All rights reserved.
+ * Use of this source code is governed by a BSD-style
+ * license that can be found in the LICENSE file.
+ *)
+
+(* https://www.rfc-editor.org/rfc/rfc3986#section-2.3
+    can also contain "." and "~" but we already have 64 characters  
+*)
+let alphabet = Base64.make_alphabet "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789-_"
```

Comment:
isn't this `Base64.uri_safe_alphabet`?

Reply:
I think so, but we might want to change it in the future, but I guess I could just do an alias now?

Reply:
Probably good to be explicit. let's keep it as is